From 9729c82d2a20ff95903e5bb7bcad15eaf0b40cd6 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 17 Sep 2017 18:41:32 -0400
Subject: [PATCH 4/5] Drop dh-autoreconf from build-depends, because it's
 automatic now

Fixes this lintian warning ->
W: btrfs-progs source: useless-autoreconf-build-depends dh-autoreconf
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 8aaf3d07..a788a888 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,6 @@ Section: admin
 Priority: optional
 Maintainer: Dimitri John Ledkov <xnox@debian.org>
 Build-Depends: debhelper (>= 10),
-               dh-autoreconf,
                e2fslibs-dev,
                pkg-config,
                libacl1-dev,
-- 
2.11.0

